src/components/profile/rockets/MyProfile.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+import '../MyProfile.scss';
+import RocketProfile from './RocketProfile';
+
+const MyProfile = () => {
+  const { rockets } = useSelector((store) => store.rockets);
+  // console.log(rockets)
```

Comment:
- [ ] **`[REQUIRED]`** Kindly check your code and remove the unnecessary portions of code commented in your codebase. It seems unprofessional in your codebase.